package.json

```diff
@@ -15,14 +15,20 @@
     "sanctuary-def": "0.6.x"
   },
   "devDependencies": {
+    "browserify": "13.1.1",
     "doctest": "0.10.x",
     "eslint": "2.9.x",
     "istanbul": "0.4.x",
     "jsverify": "0.7.x",
+    "karma": "^1.3.0",
+    "karma-browserify": "5.1.0",
+    "karma-mocha": "1.3.0",
+    "karma-sauce-launcher": "1.1.0",
     "mocha": "3.x.x",
     "remember-bower": "0.1.x",
     "sanctuary-style": "0.3.x",
     "transcribe": "0.5.x",
+    "watchify": "3.7.0",
```

Comment:
I needed it initially before Saucelabs but i can remove it now, it's only needed if you want to use watch mode for karma.